locales/es/translation.json

```diff
@@ -464,5 +480,6 @@
 	"You want to manage your diabetes data. You are caring for or supporting someone with diabetes.": "Desea administrar sus datos de diabetes. Está cuidando o apoyando a alguien con diabetes.",
 	"You've been invited to Tidepool.": "Has sido invitado a Tidepool.",
 	"Your password was changed successfully. You can now log in with your new password.": "Su contraseña fue cambiada exitosamente. Ahora puede iniciar sesión con su nueva contraseña.",
-  	"Post_submit": "Guardar"
+  	"Post_submit": "Guardar",
+  "to": "hasta"
```

Comment:
This is a duplicate key.  It's already present on line 415, where it's translated as `a`